util.py

```diff
@@ -52,6 +52,12 @@ def elideString(string):
     return string
 
 
+def splitListIntoSublists(origList, sublistLength=10):
```

Comment:
This function helps with ArcGIS' API limit of 25 people per operation.  Is it their REST API that imposes the limit or their Python library for that API?

I'm thinking if their code is like it has been in the past, it might be worthwhile to look into their Python code and see if there's a way to bypass a code-imposed limit.

Also, ESRI has sometimes been amenable to accepting feature requests, such as increasing the limit.

Reply:
It seems like the REST API, since that the limit is only mentioned there, not in any ArcGIS API pages that I've seen. Sure, a feature request would make sense, or at least a request for better documentation.

Reply:
Not sure if it would help, but `numpy.split()` or `numpy.array_split()` might do the same job as this function.  I have nothing against this function, but if we use a library function to do this, then it's one less piece of code we need to maintain.

Reply:
That's not a bad idea; I think I was trying not to change the dependencies too much. The project doesn't use `numpy` or `pandas` currently.